src/hotspot/share/gc/shenandoah/shenandoahVerifier.cpp

```diff
@@ -1354,7 +1382,18 @@ void ShenandoahVerifier::verify_rem_set_before_mark() {
 
   log_debug(gc)("Verifying remembered set at %s mark", _heap->doing_mixed_evacuations()? "mixed": "young");
```

Comment:
Might change this log message to something like:
```C++
log_debug(gc)("Verifying remembered set at %s%s mark", _heap->active_generation()->name(), _heap->doing_mixed_evacuations()? " (mixed)": "");
```

Reply:
Thanks.  Changing.